src/models/User.model.js

```diff
@@ -1,9 +1,26 @@
 'use strict';
 
+const { DataTypes } = require('sequelize');
 const { sequelize } = require('../db.js');
 
 const User = sequelize.define(
-  // your code goes here
+  'User',
+  {
+    id: {
+      type: DataTypes.INTEGER,
+      primaryKey: true,
+      autoIncrement: true,
+    },
+    name: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    },
+  },
+  {
+    tableName: 'users',
+    updatedAt: false,
+    createdAt: false,
```

Comment:
The `updatedAt` and `createdAt` fields are set to false, which means Sequelize will not automatically manage these timestamps. If you need to track when records are created or updated, consider enabling these fields by setting `timestamps: true` in the model options.